tests/debuginfo/function-arg-initialization.rs

```diff
@@ -234,6 +234,7 @@ struct BigStruct {
 
 fn non_immediate_args(a: BigStruct, b: BigStruct) {
     zzz(); // #break
+    std::hint::black_box(|| { let _ = (a, b); }); // Avoid `<optimized out>` prints
```

Comment:
Some reasonable questions might be: "Why is this needed? Why does `aarch64-gnu-llvm-20-1` optimize away the args even with `-Copt-level=0`?"

I'll look into that to make sure we're not papering over a real bug.